WcaOnRails/spec/requests/api_competitions_spec.rb

```diff
@@ -321,28 +218,29 @@ def create_wcif_events(event_ids)
 
         it "does not use CSRF protection as we use oauth token" do
           headers = { "CONTENT_TYPE" => "application/json", "ACCESS_TOKEN" => nil }
-          competition_events = [
-            {
-              id: "333",
-              rounds: [
-                {
-                  id: "333-1",
-                  format: "a",
-                  timeLimit: {
-                    centiseconds: 4242,
-                    cumulativeRoundIds: [],
-                  },
-                  cutoff: nil,
-                  advancementCondition: nil,
-                  scrambleGroupCount: 1,
-                },
-              ],
-            },
-          ]
+          competition_events = create_wcif_events(%w(333))
           patch api_v0_competition_update_events_from_wcif_path(competition), params: competition_events.to_json, headers: headers
           expect(response).to be_success
         end
       end
     end
   end
 end
+
+def create_wcif_events(event_ids)
+  event_ids.map do |event_id|
+    {
+      id: event_id,
+      rounds: [
+        {
+          id: "#{event_id}-1",
```

Comment:
Oh wait! Is this a valid round id? Shouldn't this be `#{event_id}-r1`?

Reply:
(It would be nice to add a validation on these things)

Reply:
It's not! Although it's not something I changed in this PR =) I doesn't get detected in any way as it's not really used (when saving data from WCIF) ^^

Reply:
Fixed! Ready do be merged?

Reply:
:shipit: :exclamation: